packages/@aws-cdk/aws-batch/package.json

```diff
@@ -65,18 +65,25 @@
   "devDependencies": {
     "@aws-cdk/assert": "1.14.0",
     "cdk-build-tools": "file:../../../tools/cdk-build-tools",
+    "cdk-integ-tools": "file:../../../tools/cdk-integ-tools",
     "cfn2ts": "file:../../../tools/cfn2ts",
     "pkglint": "file:../../../tools/pkglint"
   },
   "dependencies": {
+    "@aws-cdk/aws-ec2": "1.14.0",
+    "@aws-cdk/aws-ecs": "1.14.0",
+    "@aws-cdk/aws-iam": "1.14.0",
     "@aws-cdk/core": "1.14.0"
   },
   "homepage": "https://github.com/aws/aws-cdk",
   "peerDependencies": {
-    "@aws-cdk/core": "1.14.0"
+    "@aws-cdk/aws-ec2": "1.14.0",
+    "@aws-cdk/aws-iam": "1.14.0",
+    "@aws-cdk/core": "1.14.0",
+    "@aws-cdk/aws-ecs": "1.14.0"
```

Comment:
Try to have `dependencies` and `peerDependencies` in the same, alphabetical, order.

Reply:
Alright 👍